docs/model-dev-guide/api-guides/batch-processing/batch-process-api-ug.rst

```diff
@@ -151,7 +162,37 @@ example.
 
            self.output = []
 
-Step 2: Initialize the Dataset
+(Optional) Step 2: Link the Run to a Checkpoint or Model Version
+================================================================
+
+You can optionally link your batch inference run with the
+:class:`~determined.experimental.checkpoint.Checkpoint` or
+:class:`~determined.experimental.model.ModelVersion` used in the run to aggregate custom metrics on
+that object for analysis later.
+
+A full example of this can be found in the `CIFAR10 Pytorch Example
+<https://github.com/determined-ai/determined/tree/main/examples/computer_vision/cifar10_pytorch>`__
+in `inference_example.py`.
```

Comment:
[use double backticks]

To see an example, visit the ``inference_example.py`` file in the `CIFAR10 Pytorch Example
<https://github.com/determined-ai/determined/tree/main/examples/computer_vision/cifar10_pytorch>`__.

Reply:
added